magicpod-api-client.go

```diff
@@ -32,6 +32,10 @@ func main() {
 					Name:  "test_settings_number, S",
 					Usage: "Test settings number defined in the project batch run page",
 				},
+				cli.StringFlag{
+					Name:  "branch_name, br",
```

Comment:
I guess you chose `br` to avoid conflict with the existing `-B` option, but I have two concerns with this choice:

- Two characters for a short option is conventionally illegal. Some option parsers can't parse it because they recognize `-br` as turning on `b` and `r` or giving `r` to the `b` option. Does this parser allow such usage?
- Inconsistent with other short options (not only this program's options, but also many different programs')

We have two solutions:

- Giving up on assigning a short option
- Look for another short option (e.g., `-r` doesn't look used yet)

Reply:
Yeah, I missed `-B` wasn't actually used for the `batch-run` subcommand.